### DISCUSSION TITLE
Any way to add media controls in taskbar preview?

### DISCUSSION THREAD

**Discussion by AhsanSarwar45** [tauri-apps/tauri#7257] [Q&A]
I am making a music player and I would like to add media controls in the taskbar preview like in the screenshot below. Any way to do that currently?
![image](https://github.com/tauri-apps/tauri/assets/41967492/1693f919-f834-443f-ac0c-67c7a8ffe749)

**Comment by joueyl** [comment]
I'm also looking for the answer

**Comment by cardo-podcast** [comment]
it's possible?

**Reply by immorrtalz** [reply]
In windows - yep: https://learn.microsoft.com/en-us/windows/win32/shell/taskbar-extensions#thumbnail-toolbars
In tauri for now - nope...

**Reply by cardo-podcast** [reply]
Thanks. Maybe with this:
https://github.com/microsoft/windows-rs
https://microsoft.github.io/windows-docs-rs/doc/windows/index.html

is possible. Anyway  it would be great to have a tauri handler for that. Maybe on v2 :)

**Reply by immorrtalz** [reply]
Yeah, I saw that, but I don't know rust and how it works)
I think, if tauri aims to electron level, they probably need this feature to be in tauri framework

**Comment by HasibulHasan098** [comment]
Check the solution here : [https://github.com/tauri-apps/tauri/issues/10141#issuecomment-3709964745]()
